packages/x64dbg.scyllahide.vm/x64dbg.scyllahide.vm.nuspec

```
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>x64dbg.scyllahide.vm</id>
-    <version>0.0.0.20210823</version>
+    <version>0.0.0.20230324</version>
```

The tool version seems to be 1.4. The package version should be the tool version. I think it didn't have a version before that why we used version 0.0.0.date. 
```suggestion
    <version>1.4</version>
```

See https://github.com/mandiant/VM-Packages/wiki/Package-Structure#version